core/primitives/src/lib.rs

```diff
@@ -97,3 +97,14 @@ pub const MWEI: Balance = 1_000 * KWEI;
 pub const KWEI: Balance = 1_000 * WEI;
 /// 1 wei — 1
 pub const WEI: Balance = 1;
+
+/// Darwinia proposal base fee.
+pub const DARWINIA_PROPOSAL_REQUIREMENT: Balance = 5000 * UNIT;
```

Comment:
Suggest moving this to the `pallet_treasury::Config` scope. This value is much more like business logic rather than chain core primitives.